config/settings/base.py

```diff
@@ -267,6 +268,21 @@ def env(name, default=NoDefaultValue, type_=str):
 USE_TZ = True
 
 
+# Email settings
+DEFAULT_FROM_EMAIL = env("DEFAULT_FROM_EMAIL", default="no-reply@metashare.org")
```

Comment:
Should the default here at least be a real email address?

Reply:
Perhaps, but I didn't know what that might be, so I put this in as a placeholder. @davisagli do you have an alternative I could use?

Reply:
I don't want to put a real email address as the default in an open source project, no.